network-store-model/src/main/java/com/powsybl/network/store/model/ThreeWindingsTransformerAttributes.java

```diff
@@ -102,4 +102,82 @@ public Set<String> getContainerIds() {
                 .add(leg3.getVoltageLevelId())
                 .build();
     }
+
+    @Override
+    @JsonIgnore
+    public ResourceType getEquipmentType() {
+        return ResourceType.THREE_WINDINGS_TRANSFORMER;
+    }
+
+    @Override
+    @JsonIgnore
+    public List<Integer> getSideList() {
+        return List.of(1, 2, 3);
+    }
+
+    @Override
+    public LimitsAttributes getCurrentLimits(int side) {
+        if (side == 1) {
+            return leg1.getCurrentLimitsAttributes();
+        }
+        if (side == 2) {
+            return leg2.getCurrentLimitsAttributes();
+        }
+        return leg3.getCurrentLimitsAttributes();
+    }
+
+    @Override
+    public LimitsAttributes getApparentPowerLimits(int side) {
+        if (side == 1) {
+            return leg1.getApparentPowerLimitsAttributes();
+        }
+        if (side == 2) {
+            return leg2.getApparentPowerLimitsAttributes();
+        }
+        return leg3.getApparentPowerLimitsAttributes();
+    }
+
+    @Override
+    public LimitsAttributes getActivePowerLimits(int side) {
+        if (side == 1) {
+            return leg1.getActivePowerLimitsAttributes();
+        }
+        if (side == 2) {
+            return leg2.getActivePowerLimitsAttributes();
+        }
+        return leg3.getActivePowerLimitsAttributes();
+    }
+
+    @Override
+    public void setCurrentLimits(int side, LimitsAttributes limits) {
+        if (side == 1) {
+            leg1.setCurrentLimitsAttributes(limits);
+        } else if (side == 2) {
+            leg2.setCurrentLimitsAttributes(limits);
+        } else {
+            leg3.setCurrentLimitsAttributes(limits);
+        }
+    }
```

Comment:
So here, in the else, throws an exception

Reply:
Ok done